make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -219,17 +233,6 @@ ifeq ($(call isTargetOs, windows macosx), false)
       LIBAWT_XAWT_LIBS += -lpthread
     endif
 
-    ifeq ($(TOOLCHAIN_TYPE), gcc)
```

Comment:
I did not solve anything. Apparently the code has changed so the warnings is no longer triggered. Or that's at least what I assume, just as for other warnings that were no longer needed.

It might be that it also triggered only for older versions of gcc. I've tested with a couple of different versions, but I have not done an extensive test of all possible gcc version. The worst thing that can happen is that some specific, old version of gcc will still trigger these (or any other removed) warning(s). If that happens, the speedy workaround is to run with `--disable-warnings-as-error`, and then I'll restore the warning, possibly with a version check.